### DISCUSSION TITLE
UAC fails after system restart

### DISCUSSION THREAD

**Discussion by yixuan1912** [rustdesk/rustdesk#7770] [General]
### Bug Description

Hello, I encountered a problem. When the controlled computer system is restarted, the administrator rights (UAC) of the rustdesk service do not seem to work.

### How to Reproduce

1. Start the RustDesk service and grant administrator rights (UAC)
2. Restart the computer. After this, RustDesk will start automatically when the computer is turned on.
3. After the computer is restarted remotely, the administrator authority (UAC) becomes invalid.

### Expected Behavior

Maybe you can refer to the practice of sunlogin:
After starting, sunlogin will ask the user to grant administrator rights.

### Operating system(s) on local side and remote side

windows10

### RustDesk Version(s) on local side and remote side

1.2.3-1

### Screenshots

..

**Comment by xlionjuan** [comment]
Delete RustDesk
Than delete rustdesk folder at `%APPDATA%`

Reinstall again.
Doing this will also clear all config including connect history ..etc
